common/src/models/request/registerRequest.ts

```diff
@@ -2,29 +2,18 @@ import { KeysRequest } from './keysRequest';
 import { ReferenceEventRequest } from './referenceEventRequest';
 
 import { KdfType } from '../../enums/kdfType';
+import { CaptchaProtectedRequest } from './captchaProtectedRequest';
```

Comment:
Should either be in a new import group or moved up above `KeysRequest` in the import group with the other request types